### ISSUE TITLE
Convert all datasets to SWS datasets

### ISSUE THREAD

**Issue by sebastian-c** [SWS-Methodology/faoswsTrade#24]
_This is on hold until `unsdpartnersblocks` is on QA (Issue #3)_

Now that we have all the correct names for the datasets, we should convert everything in the package to SWS Datatables.
- [x] `unsdpartnersblocks` on QA
- [x] Replace package data with Datatables
- [x] Test with new tables
- [ ] Remove data from package altogether

**Comment by chrmongeau** [comment]
`unsdpartnersblocks` is on QA. I notice that the modules always use package data, but in the issue it seems that the way to go is to always use SWS datatables. Is this true?

**Comment by sebastian-c** [comment]
This is correct. I was waiting for that to be solved before we converted everything.

**Comment by chrmongeau** [comment]
With 3fa34f7, b802abb, and df50f05 the last tables read from the package (`unsdpartnerblocks`, `fclunits`, `comtradeunits`, respectively) are now read from SWS datatables.
